e2etests/testdata/sanity/child_to_child/elk/sketch.exp.svg

```diff
@@ -14,7 +14,13 @@ width="464" height="862" viewBox="-88 -88 464 862"><style type="text/css">
+</style><g id="a"><g class="shape" ><rect x="13" y="12" width="263" height="276" style="fill:#E3E9FD;stroke:#0D32B2;opacity:1.000000;stroke-width:2;" /></g><text class="text" x="144.500000" y="45.000000" style="text-anchor:middle;font-size:28px;fill:#0A0F25">a</text></g><g id="c"><g class="shape" ><rect x="12" y="398" width="264" height="276" style="fill:#E3E9FD;stroke:#0D32B2;opacity:1.000000;stroke-width:2;" /></g><text class="text" x="144.000000" y="431.000000" style="text-anchor:middle;font-size:28px;fill:#0A0F25">c</text></g><g id="a.b"><g class="shape" ><rect x="88" y="87" width="113" height="126" style="fill:#EDF0FD;stroke:#0D32B2;opacity:1.000000;stroke-width:2;" /></g><text class="text-bold" x="144.500000" y="153.000000" style="text-anchor:middle;font-size:16px;fill:#0A0F25">b</text></g><g id="c.d"><g class="shape" ><rect x="87" y="473" width="114" height="126" style="fill:#EDF0FD;stroke:#0D32B2;opacity:1.000000;stroke-width:2;" /></g><text class="text-bold" x="144.000000" y="539.000000" style="text-anchor:middle;font-size:16px;fill:#0A0F25">d</text></g><g id="(a.b -&gt; c.d)[0]"><marker id="mk-3990223579" markerWidth="10.000000" markerHeight="12.000000" refX="7.000000" refY="6.000000" viewBox="0.000000 0.000000 10.000000 12.000000" orient="auto" markerUnits="userSpaceOnUse"> <polygon class="connection" fill="#0D32B2" stroke-width="2" points="0.000000,0.000000 10.000000,6.000000 0.000000,12.000000" /> </marker><path d="M 144.000000 215.000000 L 144.000000 469.000000" class="connection" style="fill:none;stroke:#0D32B2;opacity:1.000000;stroke-width:2;" marker-end="url(#mk-3990223579)" mask="url(#labels)"/></g><mask id="labels" maskUnits="userSpaceOnUse" x="0" y="0" width="464" height="862">
+<rect x="0" y="0" width="464" height="862" fill="white"></rect>
+<rect x="136.000000" y="17.000000" width="17" height="36" fill="black"></rect>
```

Comment:
what if we tried something like semi-transparent edge lines when crossing a node label, but full opacity when it is  the edge label?
```suggestion
<rect x="136.000000" y="17.000000" width="17" height="36" fill="rgba(0,0,0,0.5)"></rect>
```

Reply:
oh that's an interesting idea. yeah i'd like to see that version @ejulio-ts

Reply:
something like this @gavin-ts @alixander 
![image](https://user-images.githubusercontent.com/102628724/205749821-3eec1ebb-7cd2-4f3e-90d1-437d40001d80.png)

Reply:
can u do this for all the test cases? just like implement it for a commit

Reply:
yeah, just wanted to confirm before working on it 😅